networkit/_NetworKit.pyx

```diff
@@ -241,6 +241,8 @@ def getMaxNumberOfThreads():
 
 def enableNestedParallelism():
 	""" Enable nested parallelism for OpenMP"""
+	from warnings import warn
+	warn("Nested parallelism has been deprecated.")
 	_enableNestedParallelism()
```

Comment:
You can remove this line, we do not allow nested parallelism anymore

Reply:
I have, and restored `__init.py` to its initial state.

Reply:
Sorry my bad that change was ok.

Reply:
OK, it's back.